tests/story.test.js

```diff
@@ -189,32 +189,33 @@ it('can create a new story in the database - createStory API', async () => {
 it('can delete a story from the database - deleteStory API', async () => {
     await storyDao.createStory(story1);
     await storyDao.createStory(story2);
-    await storyDao.createStory(story3);
-
-    await storyDao.deleteStory(story3.story_id);
+    const createdStory = await storyDao.createStory(story3);
+    
+    // await storyDao.deleteStory(story3.story_id);
```

Comment:
Can you delete this commented line? It's not required

Reply:
Removed.